### PR TITLE
Add Subscript icon

### LINKED ISSUE #431: Icon request: Subscript
## Icon Request

* Icon name: `subscript`
* Use case: Various text editors
* _Screenshots_ of similar icons: https://github.com/feathericons/feather/issues/511#issue-389037358 

I saw this icon request in original feather repo and also found it is open in https://github.com/lucide-icons/lucide/issues/119#issue-733720662 . So I brainstormed and made these two variants - 
image
image

I think first one is more in line with feathers' minimalistic design

### PR DESCRIPTION
**Pull request by nabanita-sarkar**
### Subscript icon 
image

### Icon request
Closes #431 
### Original icon request from feather
https://github.com/feathericons/feather/issues/511#issue-389037358

### CODE REVIEW

File: icons/subscript.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	viewBox="0 0 24 24" 
+	fill="none" 
+	stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+	<path d="M4 5L12 13"/>
+	<path d="M12 5L4 13"/>
+	<path d="M20 19L16 19C16 17.5 16.4417 17 17.5 16.5C18.5583 16 20 15.3338 20 14.0015C20.0007 13.5295 19.8294 13.0725 19.5163 12.7115C19.2033 12.3504 18.7688 12.1086 18.2897 12.0288C17.8106 11.949 17.3178 12.0364 16.8988 12.2755C16.4797 12.5147 16.1613 12.8901 16 13.3353"/>
```
Comment (ericfennis) [inline icons/subscript.svg:14]: Formatting and minifying path
```suggestion
  <path d="m4 5 8 8" />
  <path d="m12 5-8 8" />
  <path d="M20 19h-4c0-1.5.44-2 1.5-2.5S20 15.33 20 14c0-.47-.17-.93-.48-1.29a2.11 2.11 0 0 0-2.62-.44c-.42.24-.74.62-.9 1.07" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Nice work, small optimisations for the code to make it smaller

**Comment by nabanita-sarkar** [comment]
Thanks. I will optimize in future

**Review by ericfennis — APPROVED** [review]
Awesome!

**Comment by nabanita-sarkar** [comment]
Is there something else for me to do so that this pr can be merged?

**Comment by mittalyashu** [comment]
@ericfennis Should we also add "math" tag?

**Comment by ericfennis** [comment]
@mittalyashu I'm not exactly sure what you mean. Not the sigma (SUM) icon right?
https://lucide.dev/icon/sigma

**Pull request merged**